statistics/statistics_test.go

```diff
@@ -105,25 +104,19 @@ func TestStatisticsProcessorMetrics(t *testing.T) {
 
 			movingAverageObj := New()
 
-			_, receivedData, _ := movingAverageObj.Process("snap.gob", buf.Bytes(), nil)
-
-			var metricsNew []plugin.PluginMetricType
+			movingAverageObj.Process("snap.gob", buf.Bytes(), nil)
 
-			//Decodes the content into MetricType
-			dec := gob.NewDecoder(bytes.NewBuffer(receivedData))
-			dec.Decode(&metricsNew)
+			var metricsNew []plugin.MetricType
 			So(metrics, ShouldNotResemble, metricsNew)
```

Comment:
Rashmi and I were wondering this. The test is using a random number generator each time so we don't have a means to actually verify the value. We thought this seemed odd but wasn't sure if this was "best practice" or if there was an end-to-end test that focused on the actual values.